File: packages/protocol/src/stakes.ts
/** Shared stake presets for public ring tables and private host options. */
export interface StakePreset {
  id: string;
  label: string;
  smallBlind: number;
  bigBlind: number;
  minBuyIn: number;
  maxBuyIn: number;
}

export const STAKE_PRESETS: StakePreset[] = [
  { id: 'micro', label: 'Micro', smallBlind: 2, bigBlind: 5, minBuyIn: 100, maxBuyIn: 500 },
  { id: 'low', label: 'Low', smallBlind: 5, bigBlind: 10, minBuyIn: 200, maxBuyIn: 1000 },
  { id: 'mid', label: 'Mid', smallBlind: 10, bigBlind: 25, minBuyIn: 500, maxBuyIn: 2500 },
  { id: 'high', label: 'High', smallBlind: 25, bigBlind: 50, minBuyIn: 1000, maxBuyIn: 5000 },
];

export function stakeById(id: string): StakePreset | undefined {
  return STAKE_PRESETS.find((s) => s.id === id);
}

export const DEFAULT_STAKE_ID = 'low';

File: apps/android/core/model/src/main/java/com/pokr/android/core/model/Stakes.kt
package com.pokr.android.core.model

data class StakePreset(
    val id: String,
    val label: String,
    val smallBlind: Int,
    val bigBlind: Int,
    val buyIn: Int,
)

val STAKE_PRESETS: List<StakePreset> = listOf(
    StakePreset("micro", "Micro", 2, 5, 500),
    StakePreset("low", "Low", 5, 10, 1000),
    StakePreset("mid", "Mid", 10, 25, 2500),
    StakePreset("high", "High", 25, 50, 5000),
)

const val DEFAULT_STAKE_ID = "low"

fun stakeById(id: String): StakePreset =
    STAKE_PRESETS.find { it.id == id } ?: STAKE_PRESETS[1]
